cypress/support/commands.js

```diff
@@ -16,16 +16,26 @@
 // populate localStorage with a default connection, automatically cleared before each spec
 // assumes default 'admin' user and '' password
 // actual server URL is retrieved via ENV
-Cypress.Commands.add("connect", () => { 
+Cypress.Commands.add("connect", () => {
     // conn_val mimics actual app behavior, its value is inconsequential for establishing a connection
     let conn_val = 'admin@' + Cypress.env('API_HOST')
-    let nested = {"name":"localhost","server": Cypress.env('API_HOST'),"username":"admin","password":"","users":[],"groups":[]}
+    let nested = { "name": "localhost", "server": Cypress.env('API_HOST'), "username": "admin", "password": "", "users": [], "groups": [] }
     let obj = {}
     obj[conn_val] = nested
 
-    localStorage.setItem('connections',	JSON.stringify(obj))
+    localStorage.setItem('connections', JSON.stringify(obj))
 
- })
+})
+Cypress.Commands.overwrite('visit', (orig, url, options) => {
+    // this is a fix to include the process variable when using the Electron browser
```

Comment:
can you be more specific here, links to the relevant issues and commits would be helpful too, as we need to maintain and understand this down the road

Reply:
I added a link to the issue on Theia's repo